nextjsUIapp/src/app/about/page.tsx

```diff
@@ -1,13 +1,19 @@
 import type { Metadata } from "next";
+import {getServerSession} from "next-auth";
+import {authOptions} from "@/lib/auth";
+import {redirect} from "next/navigation";
 
 export const metadata: Metadata = {
     title:"About",
 
 };
 
-export default function About(){
+export default async function About(){
+        const current_session = await getServerSession(authOptions);
+    //user is logged in so redirect to admin page
+    if (current_session) return redirect("/admin");
     return (
-        <div>
+        <div className="set-margin-hundred">
 
             <div className="text-left">
                 <h2 className="mb-4 text-3xl font-extrabold font-extrabold leading-none text-sky-900 animate-slide-up">What is BrainKB?</h2>
```

Comment:
```suggestion
                <h2 className="mb-4 text-3xl font-extrabold font-extrabold leading-none text-white animate-slide-up">What is BrainKB?</h2>
```

Reply:
I would suggest changing all text to white since `sky-900` is hard to read on a blue background.

![image](https://github.com/user-attachments/assets/ddc62a60-cfed-4ac4-89f8-f993095390f9)

Reply:
@kabilar It's little bit strange because I do not see blue background.

This is what I see.

<img width="1508" alt="image" src="https://github.com/user-attachments/assets/1fa8e98c-2753-424b-a17f-f08c84227bee">

Reply:
It looks like the site is responsive to the theme of the operating system (i.e. light or dark mode).  When I change to light mode, I see the same color scheme in the screenshot that you provided above.  Perhaps we need to fix the background to be white regardless of the theme.  Or change the text to be responsive to the theme.